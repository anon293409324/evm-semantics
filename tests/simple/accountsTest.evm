account:
-   id: 1
-   balance: 40
-   program: PUSH 40 ; PUSH 50 ; ADD ; PUSH 0 ; PUSH 0 ; PUSH 15 ; PUSH 2 ; CALL
-   storage: 30 , 20 , 25

account:
-   id: 2
-   balance: 70
-   program: PUSH 30 ; PUSH 70 ; SUB
-   storage: 22 , 50 , 70

.Accounts


transaction:
-   to: 1
-   from: .AcctID
-   data:
-   value: 0
-   gasPrice: 1
-   gasLimit: 1000

.Transactions
