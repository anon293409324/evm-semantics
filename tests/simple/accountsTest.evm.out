<T> <k> . </k> <id> 2 </id> <pc> 3 </pc> <gas> 6 </gas> <wordStack> 40 : .WordStack </wordStack> <localMem> .Map </localMem> <gasPrice> 1 </gasPrice> <callStack> { 1 | 8 | 279 | 90 : .WordStack | .Map } .CallStack </callStack> <accounts> <account> <acctID> 2 </acctID> <program> 0 |-> ( PUSH 30 ) 1 |-> ( PUSH 70 ) 2 |-> SUB </program> <storage> 0 |-> 22 1 |-> 50 2 |-> 70 </storage> <balance> 70 </balance> </account> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 40 ) 1 |-> ( PUSH 50 ) 2 |-> ADD 3 |-> ( PUSH 0 ) 4 |-> ( PUSH 0 ) 5 |-> ( PUSH 15 ) 6 |-> ( PUSH 2 ) 7 |-> CALL </program> <storage> 0 |-> 30 1 |-> 20 2 |-> 25 </storage> <balance> 40 </balance> </account> </accounts> </T>
